test/cpp/microbenchmarks/BUILD

```diff
@@ -226,22 +226,6 @@ grpc_cc_test(
     deps = [":helpers"],
 )
 
-grpc_cc_test(
-    name = "bm_call_create",
```

Comment:
I assume the argument for getting rid of this benchmark is that it's testing mainly filter stack-based call creation, which will go away with promises?

Reply:
yeah, and so there doesn't seem to be much point in maintaining something that we don't think is useful anymore